samples/notebooks/prediction/SDK_Custom_Preprocess.ipynb

```diff
@@ -909,18 +840,215 @@
   {
    "cell_type": "markdown",
    "metadata": {
-    "id": "147a555f6c93"
+    "id": "147a555f6c93",
+    "tags": []
    },
    "source": [
     "### Run and test the container locally"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "8phJWBhe7u9u",
+    "tags": []
+   },
+   "source": [
+    "#### Set up credentials\n",
+    "\n",
+    "Setting up credentials is only required to run the custom serving container locally. Credentials set up is required to execute the `Predictor`'s `load` function, which downloads the model artifacts from Google Cloud Storage.\n",
+    "\n",
+    "To access to the Google Cloud Storage in your project, you can use one of the following:\n",
```

Comment:
To access to the Google Cloud Storage in your project -> To access Google Cloud Storage in your project

you can use one of the following -> you'll need to set up credentials by using one of the following

Reply:
Done.